resources/assets/less/variables.less

```diff
@@ -51,15 +51,12 @@
 @padding-big: (20px + @logo-size + 10px);
 
 @gutter-profile-page: @gutter-v2;
-@gutter-profile-page-desktop: @gutter-desktop;
+@gutter-profile-page-desktop: @gutter-v2-desktop;
```

Comment:
just use `@gutter-v2` and `@gutter-v2-desktop` directly? (maybe we can use css variables next time profile gutter changes again 🤔 )